shell/src/main/java/tachyon/shell/TFsShell.java

```diff
@@ -749,32 +753,20 @@ public void printUsage() {
    * @return The number of argument of the input command
    */
   public int getNumOfArgs(String cmd) {
-    if (cmd.equals("getUsedBytes")
-        || cmd.equals("getCapacityBytes")) {
+    if (cmd.equals("getUsedBytes") || cmd.equals("getCapacityBytes")
+        || cmd.equals("listLineages")) {
       return 0;
-    } else if (cmd.equals("cat")
-        || cmd.equals("count")
-        || cmd.equals("ls")
-        || cmd.equals("lsr")
-        || cmd.equals("mkdir")
-        || cmd.equals("rm")
-        || cmd.equals("rmr")
-        || cmd.equals("tail")
-        || cmd.equals("touch")
-        || cmd.equals("load")
-        || cmd.equals("fileinfo")
-        || cmd.equals("location")
-        || cmd.equals("report")
-        || cmd.equals("pin")
-        || cmd.equals("unpin")
-        || cmd.equals("free")
-        || cmd.equals("du")) {
+    } else if (cmd.equals("cat") || cmd.equals("count") || cmd.equals("ls") || cmd.equals("lsr")
```

Comment:
undo this change ... it is easier to maintain this list if there is one command per line